pep-513.rst

```diff
@@ -216,6 +211,79 @@ the list above to remove libraries that have turned out to be
 problematic or add libraries that have turned out to be safe.
 
 
+libpythonX.Y.so.1
+-----------------
+
+Note that ``libpythonX.Y.so.1`` is *not* on the list of libraries that
+a ``manylinux1`` extension is allowed to link to. It turns out that
+this is unnecessary in almost all cases: the way ELF linking works,
+extension modules that are loaded into the interpreter automatically
+get access to all of the interpreter's symbols, regardless of whether
+or not the extension itself is explicitly linked against
+libpython. Furthermore, explicit linking to libpython creates problems
+in the common configuration where Python is not built with
+``--enable-shared``. In particular, on Debian and Ubuntu systems,
+``apt install pythonX.Y`` does not even install ``libpythonX.Y.so.1``,
+meaning that any wheel that *did* depend on ``libpythonX.Y.so.1``
+could fail to import.
+
+There is one situation where extensions that are linked in this way
+can fail to work: if a host program (e.g., ``apache2``) uses
+``dlopen()`` to load a module (e.g., ``mod_wsgi``) that embeds the
+CPython interpreter, and the host program does *not* set the
+``RTLD_GLOBAL`` flag, then the embedded CPython will be unable to load
+any extension modules that do not themselves link explicitly to
+``libpythonX.Y.so.1``. Fortunately, ``apache2`` *does* set the
+``RTLD_GLOBAL`` flag, as do all the other programs that
+embed-CPython-via-a-dlopened-plugin that we could locate, so this does
+not seem to be a serious problem in practice. The incompatibility with
+Debian/Ubuntu is more of an issue than the theoretical incompatibility
+with a rather obscure corner case.
+
+This is a rather complex and subtle issue that extends beyond
+the scope of ``manylinux1``; for more discussion see: [9]_, [10]_,
+[11]_.
+
+UCS-2 vs UCS-4 builds
+---------------------
+
+All versions of CPython 2.x, plus CPython 3.0-3.2, can be built in two
+ABI-incompatible modes: one where Unicode is stored using UCS-2, and
```

Comment:
It's actually stored as UTF16 - see https://mail.python.org/pipermail/python-dev/2008-July/080892.html